tests/acceptance/features/useridentity-POST-HappyPath.feature

```diff
@@ -32,3 +32,26 @@ Feature: UserIdentity Post - Happy Path
       | P3  | P2    | P2          |
       | P3  | P2,P3 | P3          |
       | P2  | P2,P3 | P2          |
+
+  Scenario: A stored identity has a different VOT in the JWT to that stored
+    Given a user has 4 CURRENT credentials stored
+    And the user has a stored identity, with VOT "P2" in the VC and stored with "P3"
+    When I make a request for the users identity with a VTR "P3"
+    Then the status code should be 200
+    And the stored identity should be returned
+    And the stored identity isValid field is true
+    And the stored identity signatureValid field is true
+    And the stored identity kidValid field is true
+    And the stored identity should have VOT "P3"
+
+  Scenario: A stored identity has a different max_vox in the JWT to that stored
+    Given a user has 4 CURRENT credentials stored
+    And the user has a stored identity, with VOT "P2", max_vox "P3" in the VC and stored with "P4"
+    When I make a request for the users identity with a VTR "P3,P4"
+    Then the status code should be 200
+    And the stored identity should be returned
```

Comment:
I think this has now turned into a case where each assertion has become its own step. We're now documenting all the properties of the response object, rather than describing behaviour. I feel like it veers away from behaviour driven syntax. Overtime, our scenarios will become hard to maintain, brittle, too granular. Think we should try to address if possible.

Reply:
Thanks, and maybe another time we can compare the different techniques and decide what's best for us.